source/install/install-kubernetes.rst

```diff
@@ -21,22 +21,74 @@ You'll need a `Kubernetes cluster <https://kubernetes.io/docs/setup/>`__ running
 Install the operators
 ---------------------
 
-Operators are installed using ``kubectl``, and each operator is created in its own namespace. You can install and run multiple Mattermost installations in the same cluster using different namespaces.
+Nginx Ingress Controller
+^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
RST heading order: 

title: ====
H1: ----
H2: ~~~~
H3: ^^^^

These need to be H2 rather than H3, please. Also, heading syntax must be the full length of title text, else a build error is thrown.